app/views/patients/_patients.html.erb

```diff
@@ -7,6 +7,7 @@
       </select>
     </label>
     <label style="display:inline-flex;margin-bottom:0.25em">
+      <span style="white-space:nowrap;line-height:1.9;margin-right:0.375em">Scope:</span>
```

Comment:
Can you explain this one as well?

Reply:
this was one of the changes by ORAU so I am actually not sure about this one

Reply:
I'm pretty sure this view isn't even being used - we should probably delete it in a future PR (not now). I'd rather not include changes we don't understand.

Reply:
after some further investigation, it appears that this was added to match [this line](https://github.com/SaraAlert/SaraAlert/blob/master/app/views/patients/_patients.html.erb#L4) and [this line](https://github.com/SaraAlert/SaraAlert/blob/master/app/views/patients/_patients.html.erb#L17).  so I think either we leave it for consistency or remove all three

Reply:
Yeah I was wrong also - this page is the main dashboard for Enrollers which I forgot we haven't updated to a React component yet. 

This change changes it from this:
![Screen Shot 2020-12-20 at 5 17 09 PM](https://user-images.githubusercontent.com/11698457/102725767-33457280-42e7-11eb-9c1d-b80de28a7100.png)

To this:
![Screen Shot 2020-12-20 at 5 16 10 PM](https://user-images.githubusercontent.com/11698457/102725769-393b5380-42e7-11eb-865e-42a855e72830.png)

Which makes sense to me so we're all good to keep it. Resolving!